assets/src/components/routeLadder.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from "react"
+import { fetchTimepointsForRoute } from "../api"
+import { Route, Timepoint, TimepointsByRoute } from "../skate"
+import { Dispatch, setTimepointsForRoute } from "../state"
+import Loading from "./loading"
+
+interface Props {
+  route: Route
+  dispatch: Dispatch
+  timepointsByRoute: TimepointsByRoute
+}
+
+const TimepointStop = () => (
+  <div className="m-route-ladder__timepoint-stop">
```

Comment:
```suggestion
  <div className="m-route-ladder__stop">
```
BEM doesn't need the whole hierarchy. I don't think having `timepoint-` adds any understanding, so I'd lean slightly against including it.

(`__timepoint-name` does help disambiguate it from `__route-name`, so this comment wouldn't apply there.)